tests/loggers/test_wandb.py

```diff
@@ -151,6 +143,43 @@ def test_wandb_logger_dirs_creation(wandb, tmpdir):
     assert trainer.log_dir == logger.save_dir
 
 
+@mock.patch('pytorch_lightning.loggers.wandb.wandb')
+def test_wandb_log_model(wandb, tmpdir):
```

Comment:
Can we add a test for restarting an experiment ?

Reply:
I'm not sure I understand.
If you mean reloading a model, it is not done by the integration but by `wandb` api (see demo) and interact with the servers (to download the artifact). The logic of uploading a file and re-downloading the same file would probably be more specific to `wandb` itself.